pyproject.toml

```diff
@@ -294,7 +294,7 @@ warn_unused_configs = true
 ignore_missing_imports = true
 
 [tool.ruff]
-line-length = 1486
+line-length = 301
```

Comment:
isn't 301 a bit too much? maybe 120?

Reply:
That's the longest line we currently have, I didn't want to touch code in the PR, just setting guardrails against worsening the present situation

Reply:
ok, got it - makes sense